indico/htdocs/js/indico/modules/contributions/common.js

```diff
@@ -193,6 +196,48 @@
         });
     }
 
+    function setupStartDateQBubbles() {
+        $('.contrib-start-date').each(function() {
+            var $this = $(this);
+
+            $this.ajaxqbubble({
+                ajax: {
+                    url: $this.data('href')
+                },
+                style: {
+                    classes: 'qbubble-contrib-start-date'
+                },
+                onHide: function(event) {
+                    var originalEvent = event.originalEvent;
+                    var $target = $(originalEvent.target);
+                    var $this = $(this);
+
+                    // in order not to hide the qBubble when selecting a date
+                    if ((originalEvent && $target.closest('#ui-datepicker-div').length) || $target.is($this)) {
```

Comment:
I think `$target.is(this)` might work too. Also, wouldn't it make sense to include the special datepicker handling in the base ajaxqbubble code? Having a form in there is not that unlikely after all